src/actions/users.ts

```diff
@@ -484,13 +484,13 @@ export function getProfilesWithoutTeam(page: number, perPage: number = General.P
     };
 }
 
-export function getProfilesInChannel(channelId: string, page: number, perPage: number = General.PROFILE_CHUNK_SIZE, sort = ''): ActionFunc {
+export function getProfilesInChannel(channelId: string, page: number, perPage: number = General.PROFILE_CHUNK_SIZE, sort = '', options: any = {}): ActionFunc {
```

Comment:
Is there anything we could write unit tests for around this?

Reply:
Do you mean adding test cases to `/actions/users.test.js`?